assets/css/main.css

```diff
@@ -49,6 +49,57 @@ body {
   display: none;
 }
 
+/* mobile menu section */
+.mobile_menu {
+  width: 100%;
+  height: 100vh;
+  background-color: var(--dark-grey);
+  z-index: 1;
+  position: fixed;
+  top: -999px;
+  left: -999px;
+  transition: all linear;
+  transition-duration: 0.7s;
+}
+
+.mobile_menu-container {
+  height: 100%;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
- [ ] Please fix the hamburger menu is not visible on the mobile view.

- [ ] Please do your best to follow the project [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) the website is not mobile responsive

Reply:
Hello thanks for your review. I have followed all the instructions and this is how my code looks after the implementation
![mob-menu-1](https://user-images.githubusercontent.com/98595024/161724075-eee05a28-b6e7-4e94-8373-931e92d5616f.PNG)
![mob-menu-2](https://user-images.githubusercontent.com/98595024/161724220-1b007038-24af-44f6-a428-4f10075d6351.PNG)

- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.